### PR TITLE
Added github_dark colorscheme and It's preview

### PR DESCRIPTION
**Pull request by 123MayankSharma**
Hey! I have Modified the github_dark_dimmed color scheme a bit to look like github dark theme used on github's website.All the credit goes to the creator of original color scheme github_dark_dimmed @proskur1n 
and I have simply modified it a little bit for people who
prefer darker color schemes. do let me know if there any issues!

### CODE REVIEW

File: colors/github_dark.lua [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+style.text = fg
+style.caret = red
+style.accent = blue
+style.dim = fgdim
+style.divider = { common.color "#444c56" }
+style.selection = { common.color "#2e4c77" }
+style.line_number = fgdim
+style.line_number2 = fg
+style.line_highlight = {common.color "#1e202e"}
+style.scrol = fgdim
```
Comment (Guldoman) [inline colors/github_dark.lua]: `style.scrollbar`?

Comment (123MayankSharma) [reply colors/github_dark.lua]: I am extremely sorry for the typo! I have fixed it. Please have a look at it👍

Comment (Guldoman) [reply colors/github_dark.lua]: No worries!

**Comment by Guldoman** [comment]
Thank you for your contribution!

**Comment by 123MayankSharma** [comment]
> Thank you for your contribution!

You are welcome😁 and thanks for your great work on lite-xl!

**Pull request merged**